src/resources/views/fields/upload.blade.php

```diff
@@ -6,7 +6,13 @@
 	{{-- Show the file name and a "Clear" button on EDIT form. --}}
     @if (isset($field['value']) && $field['value']!=null)
     <div class="well well-sm">
-    	<a target="_blank" href="{{ isset($field['disk'])?asset(\Storage::disk($field['disk'])->url($field['value'])):asset($field['value']) }}">{{ $field['value'] }}</a>
+        @if (isset($field['disk']))
+            <a target="_blank" href="{{ (asset(\Storage::disk($field['disk'])->url((!empty($field['prefix']) ? $field['prefix'] : '').$field['value']))) }}">
```

Comment:
I get what's happening here but it's ugly...is there any way to avoid the repeated `!empty...`?

Reply:
Update done.